src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2911,7 +2913,7 @@ void NetworkOPsImp::getBookPage (
 // FIXME : support iLimit.
 void NetworkOPsImp::getBookPage (
     bool bAdmin,
-    Ledger::pointer lpLedger,
+    std::shared_ptr<ReadView const>& lpLedger,
```

Comment:
The type changed here, previous was passed by value and the current is passed by non-const reference. Was this intended?